### ISSUE TITLE
Windows Multiple IP Addresses single Network Interface [BUG]

### ISSUE THREAD

**Issue by DrEVILish** [bitfocus/companion#2876]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When starting Companion when a single Network interface is configured with 2 or more network interfaces companion is unable to select IP addresses from the NIC.

### Steps To Reproduce

1. Set multiple IPs as shown in (https://academy.showcockpit.com/tutorials/networking/multiple-ip-addresses-on-a-nic)
2. Start Companion, shows one of the IPs but is unable to start or launch the full Companion experience.

### Expected Behavior

The multiple IP addresses should be selectable and loadable from within the Companion GUI for starting the application.

### Environment (please complete the following information)

```markdown
- OS: Windows
- Browser: N/A
- Companion Version: -will update with correct info tomorrow-
```

### Additional context

I normally use RPi and MacOS for my companion instances, I don't normally use Windows for companion situations. I have done this on MacOS running 1 IP subnet for DHCP / 0.0.0.0 traffic and 1 IP Subnet for Staticly assigned equipment.

**Comment by dnmeid** [comment]
I can not reproduce your problem. Your description is not clear.
Are you running into this bug with MacOS or Windows? Which OS version exactly?
Are you using one or two network interfaces (physical adaptors)?
Which IP addresses, subnet masks, routers and so on do you have configured on your network interfaces?
Please describe exactly how your problem shows. "Shows one of the IPs but is unable to start" is not logical. If Companion shows one IP, it actually did start.

**Comment by DrEVILish** [comment]
I don't have access to the machine anymore that was causing the issues:

I am using a SINGLE - NIC
I am using 2 IP Addresses on the single NIC - linked the how to guide above.

Using Windows 10 was the one causing the issue.

IP Addresses configured were:
10.10.50.100/24
192.168.18.20/24

I've just installed a fresh Win11 VM to test this over. Unable to cause the same error as before, but only 1 IP address shows in the dropdown list.
<img width="1042" alt="Screenshot 2024-05-25 at 23 34 04" src="https://github.com/bitfocus/companion/assets/6287003/23b45b5c-12f1-48d1-b2e2-c8c137532e27">

"Shows one of the IPs but is unable to start" is logical - the "GUI" that opens starts the webserver in the background, if the webserver fails to start, then the "RUNNING" doesn't appear and cannot connect to the WebUI.

I'll update in the future if I get access to the other machine.

**Comment by dnmeid** [comment]
We are using a utility called systeminformation to get the list of network interfaces. Unfortunately it only returns the first ip on any interface.
https://github.com/bitfocus/companion/blob/c61cc19cda22793b5ef6bfe0338745ffc29bb273/launcher/main.js#L470-L480

Maybe we should change the way of obtaining the IPs.

So at least that part can be reproduced and can be called a bug, but I can't reproduce a crash or the webserver not starting with multiple IPs configured.

**Comment by DrEVILish** [comment]
Agreed, I've not been able to reproduce the crash either, If I get a chance to be hands-on with the same computer again I'll test and reproduce it was being used alongside vMix but not sure how that would cause an issue with Companion starting.
